Running tests with version of pytest >= 4.0.0 fails

The tests in Camille fails if they are run with pytest version 4.0.0 or above. This is most likely due to the 'repeat' library used in some of the tests.

Excrept from failure:
```E   _pytest.warning_types.RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.
E   Please use node.get_closest_marker(name) or node.iter_markers(name).
E   Docs: https://docs.pytest.org/en/latest/mark.html#updating-code```

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,6 +23,7 @@ before_script:
   - bandit -c bandit.yml -r camille
 
 script:
+  - pip3 install -r requirements-dev.txt
   - python3 setup.py test --verbose
 
 after_success:

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,0 +1,2 @@
+pytest==3.10.1
+pytest-repeat

